.eslintrc.json

```diff
@@ -11,7 +11,7 @@
   "extends": "eslint:recommended",
   "rules": {
     "indent": [
-      2,
+      "error",
```

Comment:
Please clarify this change

Reply:
I was attempting to see if this would fix the stickler-ci errors.  The docs for that rule (https://eslint.org/docs/rules/indent) don't seem to have any examples with an integer as the first element of the array, they are all "error".  I figured it was worth a punt :D

Reply:
2 is synonymous with error, 1 is warning, for the sake of consistency I'd like them all to be same so please change it back or change them all to "error"

Reply:
yep, @UncleClapton just explained :D - my preference is on clarity - so changing them all to "error", but I am easy either way